lib/sacastats_web/router.ex

```diff
@@ -30,10 +30,16 @@ defmodule SacaStatsWeb.Router do
   scope "/character", SacaStatsWeb do
     pipe_through :browser
 
-    get "/", CharacterController, :search
+    live "/", CharacterLive.Search
     get "/:character_name", CharacterController, :base
     get "/:character_name/:stat_type", CharacterController, :character
     live "/:character_name/sessions/:login_timestamp", SessionLive.View
+    post "/:character_name/:stat_type", CharacterController, :character_post
+    post "/:character_name/:stat_type/favorite", CharacterController, :add_favorite
+    post "/:character_name/:stat_type/unfavorite", CharacterController, :remove_favorite
+    post "/:character_name/:stat_type/:optional", CharacterController, :character_optional_post
```

Comment:
what do these `:optional` paths do?

Reply:
I think I had it there just in case there was another subpage where a character would be favorited.

Reply:
if they're not used right now and you don't have a plan to include them anywhere else soon, maybe we can remove them for now and re-add them later?

Reply:
These have been converted to more specific paths.